rfcs/0098-community-team.md

```diff
@@ -0,0 +1,190 @@
+
+The above are some suggested remedies, but these should not be taken as prescriptive. The full space of social actions is open to team members as potential tools for resolving disputes. Team members are expected to use their authority only in ways that embody justice and that are consistent with the community's values, as formalized in the code of conduct and statement of values. Team members are expected to recognize the power dynamic that they are part of, and to be proactive and explicit in making sure community members know when they're speaking in an official capacity and when they're speaking in a personal capacity. Team members are not expected to be impartial in the sense of pretending not to have personal beliefs or goals, but are expected to hold the community's values above their own, in their official actions.
+
+The community team will hold its members accountable to these ideals.
+
+## Code of Conduct and Statement of Values
+[values]: #values
+
+Following the adoption of this RFC, the code of conduct and statement of values will be displayed on a dedicated page on the NixOS website. They will read as follows, unless amended by a future RFC.
+
+The Nix community recognizes that its members come from many walks of life, and that it is natural for us to disagree on some things. Nonetheless, we are all committed to certain values, to allow us to work together and build awesome technologies.
+
+Community members are expected to follow these values in their interactions in all official Nix social spaces, including websites, chat rooms, bug threads, and conferences. We have a community team who will set and enforce social norms, to make that easier for everyone. faCommunity members are expected to follow the community team's requests.
+
+If you have opinions on the community team's work, get involved! It's a rotating team, and we want to develop the leadership potential in all community members.
+
+Our values are as follows.
+
+**Collaboration** — We all want to make the best technologies we can. We work together on our common goals, with open discussion, and without petty infighting or focusing on blame. We are all, always, teaching each other to be better.
+
+**Communication** — When we encounter problems, we focus on solving them, not on fighting each other. When we disagree on technical subjects we follow appropriate processes to escalate them to technical leadership. We try to avoid and de-escalate personal conflicts, but when they are unavoidable we seek mediation.
```

Comment:
Simpler?
```
**Communication** — Problems are solved by collaboration and escalated appropriately.
```

Reply:
We're not trying to be simple for the sake of simplicity. The statement of values is the most important normative part of the document, referenced frequently from elsewhere. I'm aware that the wording of that section is a bit long, but there's a reason for every single word and we're not accepting changes that merely remove substance without capturing the concern somehow.

Reply:
It's exactly because the statement of values is so important that I've been focusing on it. My intention is to capture the concern as simply as possible in order to be clear and make them applicable as guidelines for decision making.

For this, I guess it is a bit strange to elaborate on communication by focusing on problems. Or at least that may not be the ideal word here. How about?

```
**Communication** — We build technology through discussion, focusing on solving problems, and appropriate escalation of disagreements.
```

errr... i'm not super pleased with this one, but I'm thinking it's in the right direction, thoughts?

random sidebar: I've found too much agreement to be sometimes as stagnant as too much disagreement is stifling. There's a healthy middle ground somewhere in between.

Reply:
Thanks. I do welcome your efforts, I am glad we're on the same page that it matters. I also know that my wording is inelegant. I just wanted to nudge you in the direction of not taking things out just for the sake of shortening it.

Heh, you're right that the current wording may be overly negative. The reality is that conflict happens so we do need to have ways to resolve it; I wouldn't want the statement of values to be written as if for a world where conflict never comes up in the first place.

The biggest thing that your "appropriate escalation of disagreements" fails to capture is that it doesn't describe *what* escalation is appropriate. The current wording gives specific advice on what channels to go through, breaking it into two cases: With technical issues, escalate to technical leadership. With personal issues, de-escalate (the opposite course of action to technical issues!) when it's possible, and get mediation when it isn't.

Reply:
I second: value statements are to capture values.

Neither rules, nor anything else. Acknowledign disagreememt is fine, necause, obviously those values are rooted in a specific context.

In general, the wording is too negative to be really engaging in a broad range of situations, on the other hand it leds very specificeaning to very specific situations.

That level of interpretation, though, should be handed over entirely to the team and not be part of this document.

Reply:
The thought process is that, since it's a rotating team without permanent members, the values statement does have to be concrete enough to guide future team members to understand what their job is. I don't want it to be too concrete, and I know the wording in this section is awkward, but I do think that both positive ideals and negative examples need to be part of it.

Reply:
(Just so other readers know: This thread of discussion is continuing via chat, any conclusions will be summarized here. I acknowledge the point about my argument exemplifying distrust of the community team, which does sound like a good reason to reconsider.)